src/OrbitQt/orbitsamplingreport.ui

```diff
@@ -120,13 +120,16 @@
                <string/>
               </property>
               <property name="icon">
-               <iconset resource="../icons/orbiticons.qrc">
+               <iconset>
                 <normaloff>:/actions/keyboard_arrow_right</normaloff>:/actions/keyboard_arrow_right</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="CallstackLabel">
+              <property name="toolTip">
```

Comment:
I feel like this tooltip is very hidden as it's a tooltip for a label, I would not really expect that. But then again, all our tooltips are fairly hidden I guess. One option would be to add a dedicated label (e.g., below the existing one with a smaller font). Just some random thought.